json/UI_Text.txt

```diff
@@ -23947,192 +23947,192 @@
 	},
 	{
 		"assign": "survival_return_dialog_return_button",
 		"jp_text": "再開する",
-		"tr_text": ""
+		"tr_text": "Resume"
 	},
 	{
 		"assign": "survival_return_dialog_end_button",
 		"jp_text": "終了する",
-		"tr_text": ""
+		"tr_text": "Exit"
 	},
 	{
 		"assign": "survival_quest_name",
 		"jp_text": "極討サバイバル",
-		"tr_text": ""
+		"tr_text": "Extreme Survival"
 	},
 	{
 		"assign": "survival_quest_ranking_no_entry",
 		"jp_text": "<color=#e25f04>未参加</color>",
-		"tr_text": ""
+		"tr_text": "<color=#e25f04>N/A</color>"
 	},
 	{
 		"assign": "sb_reward_dialog_t",
 		"jp_text": "極討サバイバル報酬",
-		"tr_text": ""
+		"tr_text": "Extreme Survival Reward"
 	},
 	{
 		"assign": "sb_reward_dialog_score_title_caption",
 		"jp_text": "討伐数・獲得スコア・ボーナス",
-		"tr_text": ""
+		"tr_text": "Defeated Enemies Score Bonus"
 	},
 	{
 		"assign": "sb_reward_dialog_killcount_caption",
 		"jp_text": "討伐数",
-		"tr_text": ""
+		"tr_text": "Defeated Enemies"
 	},
 	{
 		"assign": "sb_reward_dialog_score_caption",
 		"jp_text": "スコア",
-		"tr_text": ""
+		"tr_text": "Score"
 	},
 	{
 		"assign": "sb_reward_dialog_reward_title_caption",
 		"jp_text": "報酬",
-		"tr_text": ""
+		"tr_text": "Reward"
 	},
 	{
 		"assign": "sb_reward_dialog_phase_bonus_title_caption",
 		"jp_text": "フェーズ達成ボーナス",
-		"tr_text": ""
+		"tr_text": "Phase Reached Bonus"
 	},
 	{
 		"assign": "sb_reward_dialog_phase_bonus_num",
 		"jp_text": "+{0}個",
-		"tr_text": ""
+		"tr_text": "+{0} Phases"
 	},
 	{
 		"assign": "SB_shop_dialog_t",
 		"jp_text": "【PHASE{0}】 使用効果選択",
-		"tr_text": ""
+		"tr_text": "【PHASE{0}】 Choose an effect"
 	},
 	{
 		"assign": "SB_shop_dialog_Caption",
 		"jp_text": "使用する効果を1つ選んでください。",
-		"tr_text": ""
+		"tr_text": "Please choose 1 effect."
 	},
 	{
 		"assign": "SB_shop_PhaseStart_ButtonCaption",
 		"jp_text": "<color=#FFFF00>【PHASE{0}】を開始</color>",
-		"tr_text": ""
+		"tr_text": "<color=#FFFF00>【PHASE{0}】Start</color>"
 	},
 	{
 		"assign": "SB_shop_Use_ButtonCaption",
 		"jp_text": "　　　効果を使用",
-		"tr_text": ""
+		"tr_text": "　　　Use an effect"
 	},
 	{
 		"assign": "SB_status_dialog_t",
 		"jp_text": "使用効果・ステータス",
-		"tr_text": ""
+		"tr_text": "Effect status"
 	},
 	{
 		"assign": "SB_status_LTBuff_TitleCaption",
 		"jp_text": "持続効果",
-		"tr_text": ""
+		"tr_text": "Persisting Effect"
 	},
 	{
 		"assign": "SB_status_STBuff_TitleCaption",
 		"jp_text": "時限効果",
-		"tr_text": ""
+		"tr_text": "Periodic Effect"
 	},
 	{
 		"assign": "SB_shop_point_possess",
 		"jp_text": "所持サポートポイント <color=#68a0a7>/</color>",
-		"tr_text": ""
+		"tr_text": "Owned Support Points <color=#68a0a7>/</color>"
 	},
 	{
 		"assign": "SB_shop_point_use",
 		"jp_text": " 消費<color=#68a0a7> /</color>",
-		"tr_text": ""
+		"tr_text": " Cost<color=#68a0a7> /</color>"
 	},
 	{
 		"assign": "SB_Spoint_dialog_t",
 		"jp_text": "【PHASE{0}】 終了",
-		"tr_text": ""
+		"tr_text": "【PHASE{0}】 End"
```

Comment:
【】are just japanese quotes.

Reply:
oh, I know that but I wasn't sure if they were also used for variable stuff since in the screenshot where it says Phase 1 Start it doesn't show 【】